probe-rs/src/debug/variable.rs

```diff
@@ -751,7 +772,7 @@ impl Variable {
                 }
             } else {
                 // We don't have a value, and we can't generate one from children values, so use the type_name
-                self.type_name.to_string()
+                format!("{:\t<indentation$}{}", "", self.type_name.to_string())
```

Comment:
Please fix the lint, to_string() is not necessary here.

Reply:
Done :)

Reply:
Done